src/Take.Blip.Builder/Variables/ResourceVariableProviderBase.cs

```diff
@@ -37,7 +37,7 @@ public async Task<string> GetVariableAsync(string name, IContext context, Cancel
 
                 if (resourceCommandResult.Status != CommandStatus.Success)
                 {
-                    _logger.Warning($"Variable {name} from {_resourceName} not found");
+                    _logger.Warning("Variable {Name} from {ResourceName} not found", name, _resourceName);
```

Comment:
```suggestion
                    _logger.Warning("Variable {VariableName} from {ResourceName} not found", name, _resourceName);
```

Use `VariableName´, since `Name` is too generic.